platform/os_windows.go

```diff
@@ -23,6 +23,9 @@ const (
 	// CNIRuntimePath is the path where CNI state files are stored.
 	CNIRuntimePath = ""
 
+	// CNILockPath is the path where CNI lock files are stored.
+	CNILockPath = ""
```

Comment:
Should we make this also in separate dir?

Reply:
this is windows and can update this when required. i leave it to current behaviour